src/components/scenes/FioSentRequestDetailsScene.js

```diff
@@ -0,0 +1,131 @@
+  selectedFioSentRequest: any,
+  isoFiatCurrencyCode: any,
+  exchangeRates: ExchangeRatesState
+}
+
+export type FioSentRequestDetailsDispatchProps = {}
+
+type Props = FioSentRequestDetailsProps & FioSentRequestDetailsDispatchProps
+
+type LocalState = {
+  memo: string,
+  focused: boolean
+}
+
+export class FioSentRequestDetailsComponent extends Component<Props, LocalState> {
+  constructor (props: Props) {
+    super(props)
+    const state: LocalState = {
+      memo: '',
+      focused: false
+    }
+    this.state = state
+  }
+
+  fiatAmount = (currencyCode: string, amount: string) => {
+    const { exchangeRates, isoFiatCurrencyCode } = this.props
+    let fiatPerCrypto
+    if (currencyCode === Constants.FIO_STR) {
+      fiatPerCrypto = 1
+    } else {
+      const rateKey = `${currencyCode}_${isoFiatCurrencyCode}`
+      fiatPerCrypto = exchangeRates[rateKey] ? exchangeRates[rateKey] : 0
+    }
+    const amountToMultiply = parseFloat(amount)
+
+    return (fiatPerCrypto * amountToMultiply).toFixed(2)
+  }
+
+  amountField = (amount: string, currencyCode: string, cryptoSymbol: string, currencySymbol: string) => {
+    return (
+      <View style={styles.row}>
+        <T style={styles.title}>
+          Amount: {amount} {cryptoSymbol} ({currencySymbol}
+          {this.fiatAmount(currencyCode, amount)})
+        </T>
+      </View>
+    )
+  }
+
+  requestedField = (payer: string, status: string) => {
+    let statusLabel = (
+      <T style={styles.title}>
+        {s.strings.title_fio_requested} {s.strings.fragment_transaction_list_sent_prefix}
+        {s.strings.word_to_in_convert_from_to_string}
+      </T>
+    )
+    if (isSentFioRequest(status)) {
+      statusLabel = <T style={[styles.title, styles.titleReceived]}>{s.strings.fragment_transaction_list_receive_prefix}</T>
+    }
+    if (isRejectedFioRequest(status)) {
+      statusLabel = <T style={[styles.title, styles.titleRejected]}>{s.strings.fio_reject_status}</T>
+    }
+    return (
+      <View style={styles.row}>
+        {statusLabel}
+        <T style={styles.title}>{payer}</T>
+      </View>
+    )
+  }
+
+  memoField = (memo: string) => {
+    if (memo) {
+      return (
+        <View style={styles.row}>
+          <T style={styles.title}>Memo</T>
+          <T style={styles.title}>{memo}</T>
+        </View>
+      )
+    }
+
+    return <View style={styles.row} />
+  }
+
+  dateField = (date: Date) => {
+    return (
+      <View style={styles.row}>
+        <T style={styles.title}>{s.strings.fio_date_label}</T>
+        <T style={styles.title}>{intl.formatExpDate(date, true)}</T>
+      </View>
+    )
+  }
+
+  render () {
+    return (
+      <SceneWrapper>
+        <SafeAreaView>
+          <View>
+            {this.amountField(
+              this.props.selectedFioSentRequest.content.amount,
+              this.props.selectedFioSentRequest.content.token_code,
+              this.props.selectedFioSentRequest.content.token_code,
+              '$'
```

Comment:
$ Should not be hardcoded but looked up using the account currency code

Reply:
changed in last commit